### PR TITLE
fix: admin view pagination

### PR DESCRIPTION
**Pull request by juniusfree**
**demo:** https://www.loom.com/share/67fe7177c1ed47e98de40999687b2bb3

### CODE REVIEW

File: wondrous-app/components/Common/KanbanBoard/styles.ts [hunk 1]
```diff
@@ -12,7 +12,6 @@ export const KanbanBoardContainer = styled.div`
   margin-top: 32px;
 `;
 export const LoadMore = styled.div`
-  height: 50px;
```
Comment (Lamperoyge) [inline wondrous-app/components/Common/KanbanBoard/styles.ts:15]: if we remove this does the org board / pod boar kanban view has the same load more functionality?

File: wondrous-app/components/Dashboard/boards/index.tsx [hunk 2]
```diff
@@ -1,13 +1,13 @@
...
+import { useMe } from 'components/Auth/withAuth';
+import Boards from 'components/Common/Boards';
+import BoardsActivity from 'components/Common/BoardsActivity';
+import { FilterItem, FilterItemIcon, FilterItemName } from 'components/Common/Filter/styles';
+import CreateDaoIcon from 'components/Icons/createDao';
+import CreatePodIcon from 'components/Icons/createPod';
 import {
   GET_JOIN_ORG_REQUESTS,
   GET_JOIN_POD_REQUESTS,
-  GET_PER_STATUS_TASK_COUNT_FOR_USER_BOARD,
```
Comment (Lamperoyge) [inline wondrous-app/components/Dashboard/boards/index.tsx:10]: afaik this query fetches the counts for tasks in user board, do we still update that if we don't use this?

**Review by Lamperoyge — COMMENTED** [review]
left two comments, other than this lgtm

**Review by vefimov — APPROVED** [review]

File: wondrous-app/components/Common/KanbanBoard/styles.ts (hunk 1 above)
Comment (juniusfree) [reply wondrous-app/components/Common/KanbanBoard/styles.ts:15]: @Lamperoyge Yes, it still works.

File: wondrous-app/components/Dashboard/boards/index.tsx (hunk 2 above)
Comment (juniusfree) [reply wondrous-app/components/Dashboard/boards/index.tsx:10]: @Lamperoyge Let me know if this answers your question or not.

No, the task counts on the panel won't be updated if we don't use `GET_PER_STATUS_TASK_COUNT_FOR_USER_BOARD`.

image

**Review by Lamperoyge — APPROVED** [review]

**Pull request merged**